test/serialization/DefaultSerializersLiveTest.js

```diff
@@ -160,5 +162,30 @@ stringSerializationPolicies.forEach(function(stringSerializationPolicy) {
                 return expect(response.result.toString()).to.equal('\u0040\u0041\u01DF\u06A0\u12E0\u{1D306}');
             });
         });
+
+        it('rest value', function () {
+            // Making sure that the object is properly de-serialized at the server
+            var restValue = new RestValue();
+            restValue.value = Buffer.from('{"test":"data"}').toJSON().data;
+            restValue.contentType = Buffer.from('text/plain').toJSON().data;
+
+            var script = 'var map = instance_0.getMap("' + map.getName() + '");\n' +
```

Comment:
nit: if we want to make sure that the stored `RestValue` is correctly deserialized on member side, wouldn't it be simpler to read the value via IMDG REST API after we put it in the map?

Reply:
Yes thats also an option, but I found it simpler to write using remote controller script. I could change that if you want

Reply:
In my opinion, an HTTP request instead of a script will make this test more readable. But that's a matter of taste, so it's up to you to change it or not.